### PR TITLE
members: add css rules

### PR DESCRIPTION
**Pull request by jrcastro2**
* closes https://github.com/inveniosoftware/invenio-communities/issues/458
* closes https://github.com/inveniosoftware/invenio-communities/issues/456

### CODE REVIEW

File: invenio_app_rdm/theme/assets/semantic-ui/less/invenio_app_rdm/theme/collections/grid.overrides [hunk 1]
```diff
@@ -85,6 +85,10 @@
     padding-top: 5px;
     padding-bottom: 5px;
   }
+
+  .column.searchbar-with-filters{
```
Comment (kpsherva) [inline invenio_app_rdm/theme/assets/semantic-ui/less/invenio_app_rdm/theme/collections/grid.overrides:89]:
```suggestion
  .flex-container {
```
I am trying to remove all the naming which is connected with our elements of the interface, and replace them with the semantic way of naming, which is based on the actual "behaviour", which should reduce the amount of custom CSS - `flex-container` becomes reusable not only for search bar filters. it should be also placed in site.overrides

File: invenio_app_rdm/theme/assets/semantic-ui/less/invenio_app_rdm/theme/elements/input.overrides [hunk 2]
```diff
@@ -1,3 +1,7 @@
 /***********************************************
          Invenio App RDM Input Overrides
 ***********************************************/
+
+.ui.input.searchbar-with-filters{
```
Comment (kpsherva) [inline invenio_app_rdm/theme/assets/semantic-ui/less/invenio_app_rdm/theme/elements/input.overrides:5]: similar comment as above

File: invenio_app_rdm/theme/assets/semantic-ui/less/invenio_app_rdm/theme/globals/site.overrides [hunk 3]
```diff
@@ -433,3 +433,11 @@ dl.details-list {
   margin-left: auto;
 }
 
+.flex-container {
+  display: flex !important;
+}
+
+
+.contents-container {
```
Comment (kpsherva) [inline invenio_app_rdm/theme/assets/semantic-ui/less/invenio_app_rdm/theme/globals/site.overrides:441]: what is this class needed for ?

File: invenio_app_rdm/theme/assets/semantic-ui/less/invenio_app_rdm/theme/views/item.overrides [hunk 4]
```diff
@@ -68,15 +68,14 @@
...
-
 /*Description*/
 .item div.description:not(.license-item-description) {
   display: initial !important;
 }
 
+.ui.items {
+  .item.members-dopdown-option {
+    white-space: initial;
+    min-width: 350px;
```
Comment (kpsherva) [inline invenio_app_rdm/theme/assets/semantic-ui/less/invenio_app_rdm/theme/views/item.overrides:79]: please use relative units

File: invenio_app_rdm/theme/assets/semantic-ui/less/invenio_app_rdm/theme/views/item.overrides [hunk 5]
```diff
@@ -68,15 +68,14 @@
...
 .item div.description:not(.license-item-description) {
   display: initial !important;
 }
 
+.ui.items {
+  .item.members-dopdown-option {
+    white-space: initial;
+    min-width: 25em;
+  }
+}
```
Comment (Pineirin) [inline invenio_app_rdm/theme/assets/semantic-ui/less/invenio_app_rdm/theme/views/item.overrides:81]: Is it ok to not leave a new line at the end of the css or we should?

File: invenio_app_rdm/theme/assets/semantic-ui/less/invenio_app_rdm/theme/globals/site.overrides (hunk 3 above)
Comment (kpsherva) [reply invenio_app_rdm/theme/assets/semantic-ui/less/invenio_app_rdm/theme/globals/site.overrides:444]: I meant more what kind of a problem does it solve, sorry for not being specific. We noticed also yesterday that the elements need to be aligned vertically in the middle (filters are a bit misaligned with the search bar

**Pull request merged**